test/e2e/specs/wp-signup-spec.js

```diff
@@ -24,7 +24,6 @@ import CreateYourAccountPage from '../lib/pages/signup/create-your-account-page.
 import CheckOutPage from '../lib/pages/signup/checkout-page';
 import ImportFromURLPage from '../lib/pages/signup/import-from-url-page';
 import SiteTypePage from '../lib/pages/signup/site-type-page';
-import SiteTopicPage from '../lib/pages/signup/site-topic-page';
```

Comment:
With this not needed and not used anywhere else, you could also get rid of `site-topic-page.js`.

Reply:
Fixed.